starlette/requests.py

```diff
@@ -18,8 +18,30 @@ class ClientDisconnect(Exception):
     pass
 
 
-class State:
-    pass
+class State(object):
+    def __init__(self, state={}):
+        self._state = state
+
+    def __setattr__(self, key, value: typing.Any) -> None:
+        if key == '_state':
+            super(State, self).__setattr__(key, value)
+        else:
+            self._state[key] = value
+
+    def __getattr__(self, key) -> typing.Any:
+        if key == '_state':
+            super(State, self).__getattr__(key)
```

Comment:
This is the single line missing coverage ATM

Reply:
Done. 100% coverage! 💯💥